lib/defaults/config.json

```diff
@@ -102,7 +102,8 @@
     "sourcePath": "js",
     "webpack": {
       "enable": false,
-      "bundles": []
+      "bundles": [],
+      "verbose": false
```

Comment:
document this in the README as well

Reply:
i think you should also take a stab at writing a couple automated tests for this feature too